packages/webapp/src/assets/colors.scss

```diff
@@ -80,13 +80,17 @@
   --Colors-Secondary-Secondary-green-100: #d5eee1;
   --Colors-Secondary-Secondary-green-200: #c1e6d2;
   --Colors-Secondary-Secondary-green-400: #93d4b1;
+  --Colors-Secondary-Secondary-green-500: #F2FAF5;
```

Comment:
I think @kathyavini knows an extension we use to keep these all lowercase

Reply:
Thank you!